.github/workflows/pull-request-automation.yml

```diff
@@ -0,0 +1,15 @@
+on: pull_request
+name: Pull request automation
+
+jobs:
+  pull-request-automation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - run: npm install
+        env:
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+      - run: npm run build:packages
```

Comment:
I could eliminate this step by not using Babel transformations for the action. Am thinking it's worth it—`import` syntax is the only thing we're using that Node doesn't have by default.

Reply:
Agreed, probably not worth keeping it just for `import`.

Reply:
Removed Babel compilation in 3c22f92 👌